kyo-data/shared/src/main/scala/kyo/Frame.scala

```diff
@@ -72,7 +72,7 @@ object Frame:
         def show: String = s"Frame(${Position.show(position)}, $className, $methodName, $snippetShort)"
 
         def render: String =
-            Ansi.highlight(s"// ${Position.show(position)} $className $methodName", snippet.toString(), s"", Position.lineNumber(self))
+            Ansi.highlight(s"// ${Position.show(position)} $className $methodName", snippet.toString(), s"", Position.lineNumber(self) - 2)
```

Comment:
The start line was incorrect here because the macro captures code around the actual line